qemu/tests/nic_opt.py

```diff
@@ -0,0 +1,154 @@
+import os
+import logging
+import time
+
+from virttest import error_context
+from virttest import utils_net
+from virttest import utils_netperf
+from virttest import utils_misc
+from virttest import utils_test
+from virttest import data_dir
+from avocado.utils import cpu
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    NIC option check test.
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    def test_netperf():
+        """
+        Netperf stress test for nic option.
+        """
+        netperf_client_link = os.path.join(data_dir.get_deps_dir("netperf"),
+                                           params.get("netperf_client_link"))
+        client_path = params.get("client_path")
+
+        n_client = utils_netperf.NetperfClient(
+            vm.get_address(), client_path, params.get("pkg_md5sum"),
```

Comment:
No,if delete it,we will get an error at runtime

Reply:
Because also need to modify `netperf_client_link` to `netperf_source=netperf_client_link`
```python
    def __init__(self, address, netperf_path, md5sum="", netperf_source="",
                 client="ssh", port="22", username="root", password="123456",
                 prompt="^root@.*[\#\$]\s*$|", linesep="\n", status_test_command="echo $?"):
```

Reply:
thanks,I have replaced it.